File: java14/iniciante/1002.java
import java.io.IOException;
import java.util.Scanner; 
 
public class Main {
 
    public static void main(String[] args) throws IOException {
 
        double raio;
        double pi = 3.14159;

        Scanner scan = new Scanner(System.in);
        raio = scan.nextDouble();

        System.out.printf("A=%.4f\n", pi * Math.pow(raio, 2));
 
    }
 
}

File: go/iniciante/1002.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var r, pi, a float64
	pi = 3.14159
	fmt.Scanf("%f", &r)
	a = pi * math.Pow(r, 2)
	fmt.Printf("A=%.4f\n", a)
}
